contracts/StorageContract.sol

```diff
@@ -147,39 +147,53 @@ abstract contract StorageContract is DecentralizedKV {
 
     /// @notice Checks the payment using the last mine time.
     function _prepareAppendWithTimestamp(uint256 _timestamp, uint256 _batchSize) internal {
```

Comment:
I believe the function names like prepareAppend... are no longer appropriate, as they actually execute after the appending process